src/freenet/node/NodeDispatcher.java

```diff
@@ -846,7 +846,7 @@ public void run() {
 	private boolean handleRoutedRejected(Message m) {
 		if(!node.enableRoutedPing()) return true;
 		long id = m.getLong(DMT.UID);
-		Long lid = Long.valueOf(id);
+		Long lid = id;
```

Comment:
We don’t need `lid` if we can use `id`, right?

Reply:
Fixed